wscl-issues/draft/boolean-definition

```diff
@@ -0,0 +1,68 @@
+Issue:          BOOLEAN-DEFINITION
+Forum:          Cleanup
+Category:       CLARIFICATION
+Status:         draft
+Edit History:   16-May-24, Version 1 by Jin-Cheng Guu
+References:     BOOLEAN
+
+Problem Description:
+
+  In the draft ANSI Common Lisp specification, the description of the type
+  BOOLEAN [a] can be more accurate.
+
+    [a]: The type boolean contains the symbols t and nil, [..]
+
+  It is more accurate to use the word "consists of" than "contains". After
+  all, according to the glossary [b], BOOLEAN is exclusively either the symbol
+  T or the symbol NIL.
+
+    [b]: boolean n. an object of type boolean; that is, one of the following
+    objects: the symbol t (representing true), or the symbol nil (representing
+    false).
+
+Proposal:
+
+  This proposal changes the description of the type BOOLEAN so that [a]
+  instead reads:
+
+    > The type boolean consists of the symbols t and nil, [..]
+
+Test Cases:
+
+  N/A (Test cases are difficult to provide, because conforming implementations
```

Comment:
One test case would be `(subtypep 'boolean '(member t nil))`. On a conforming implementation this could return `T NIL` if the proposal applies, `NIL T` if it doesn't, or `NIL NIL` for no information either way. Better than nothing.

Reply:
Thanks for the suggestion. In the spec though it says 
> The second value is always true when the first value is true.
So a conforming implementation should return `T T` instead of `T NIL`.

Both `sbcl-2.4.1` and `ecl-23.9.9` return `T T`.